_includes/configuration/toolbar-mode.md

```diff
@@ -6,6 +6,25 @@
 
 The `toolbar_mode` option is used to extend the toolbar to accommodate the overflowing toolbar buttons. This option is useful for small screens or small editor frames.
 
+The editor will move toolbar buttons to the toolbar drawer based on:
+
+- The defined toolbar groups.
+- The width of the editor.
+
+For example. If the following toolbar configuration is wider than the editor, the toolbar will be split at the group separator (`|`).
```

Comment:
"toolbar will be split" - is that clear enough? would we want to be explicit and say something like "the items before the separator will appear in the main toolbar and the rest will be moved to the drawer"?

Reply:
May of been fixed by: https://github.com/tinymce/tinymce-docs/pull/1612/commits/85c06bb5d09d833d450f339a14642ad85c3ad1db